src/enum.cr

```diff
@@ -10,7 +10,15 @@
 # end
 # ```
 #
-# Values start with the value `0` and are incremented by one, but can be overwritten.
+# Values start with the value `0` and are incremented by one, but can be overwritten:
+#
+# ```
+# enum Color
+#   Red   = 5
+#   Green = 10
```

Comment:
ah nvm...this messes up the example on the following line. might not need this PR then...